Algorithm/DP/readme.md

```diff
@@ -45,5 +45,6 @@ be avoided by constructinng a temporary array dp[] and memoizing the computed va
 
 - Climbing Stairs ----> [C++](/Code/C++/Climbing_Stairs.cpp)
 
```

Comment:
Can you please remove these extra lines. Just to make it consistent with other readme files.